mk/config.mk

```diff
@@ -363,6 +363,13 @@ CFG_TA_FTRACE_SUPPORT ?= n
 #     display it in milliseconds
 CFG_FTRACE_US_MS ?= 10000
 
+# Core syscall function tracing.
+# When this option is enabled, OP-TEE core is instrumented with GCC's
+# -pg flag and will output syscall function graph in user TA ftrace
+# buffer
+CFG_CORE_SYSCALL_FTRACE ?= n
```

Comment:
Minor: `CFG_SYSCALL_FTRACE` maybe?

Reply:
Okay will do.

I have one related query here, when do we generally use `*_TA_*` and `*_CORE_*` in config option names as current usage seems a bit confusing?

Reply:
No clear hard rule here I'm afraid :-/ I'd say we should use `_CORE` and `_TA` to designate features that are specifically for the core or for TAs but not both.
For this very config: `CFG_CORE_SYSCALL_FTRACE`, the reason why we can drop `_CORE` is because `_SYSCALL` is already implicitly a core (kernel) thing.
In the end it's down to personal preference but we should always make sure the names are precise, unambiguous, and preferably short. Your proposed name is fine on the two first aspects and might just be improved on the last one ;-)

Reply:
Sounds reasonable, will update.